string rgbBaseFileName
int64 areaNum
area[] listAreas
---
bool ok
pcArea[] listPcAreas
